WcaOnRails/config/routes.rb

```diff
@@ -348,11 +348,11 @@
       end
       get '/records' => "api#records"
 
-      resources :roles, only: [:create, :show, :update, :destroy]
+      resources :user_roles, only: [:create, :show, :update, :destroy]
       scope 'roles' do
```

Comment:
Shouldn't this be migrated to `scope user_roles` as well?

Reply:
On that note: You can open the `do` block directly after the `resources`:

```ruby
resources :user_roles, only: [...] do
  get '/sub-route'
end
```

Reply:
> Shouldn't this be migrated to `scope user_roles` as well?

Oh yes, thanks for pointing that. I've made the change. Glad that I didn't had to change anywhere other than that particular line.



> On that note: You can open the `do` block directly after the `resources`:
> 
> ```ruby
> resources :user_roles, only: [...] do
>   get '/sub-route'
> end
> ```

If I do that, the route will be like `user_roles/{id}/sub-route` and we don't have any `id` to give since the APIs are fetching a list of roles and not working on a single role.

This worked for `wrt/reset_claim_count` because there we had a `person_id` to work on.